core/internal/cltest/postgres.go

```diff
@@ -46,7 +48,15 @@ func createPostgresChildDB(tc *TestConfig, originalURL string) func() {
 }
 
 func createTestDB(t testing.TB, parsed *url.URL) *url.URL {
-	dbname := fmt.Sprintf("%s_%s", parsed.Path[1:], models.NewID().String())
+	dbname := fmt.Sprintf("%s_%s_%s", parsed.Path[0:], models.NewID().String()[:8], t.Name())
+	dbname = strings.Replace(dbname, "/", "_", -1)
+	dbname = strings.Replace(dbname, "Test", "", 1)
+	dbname = strings.ToLower(dbname)
+	var safeCharsOnly = regexp.MustCompile(`[^0-9a-z_]+`)
+	dbname = safeCharsOnly.ReplaceAllString(dbname, "")
+	if len(dbname) > 63 {
+		dbname = dbname[:63]
+	}
```

Comment:
Maybe update this to correctly work with runes?  I understand there's the regex above, but its good to consistently handle strings in a safe manner.  They are not straightforward in Go.  Check -> https://stackoverflow.com/questions/15018545/how-to-index-characters-in-a-golang-string

Reply:
I'm not sure postgres even supports non-ascii in table names. Since everything will be ASCII after the regex, I think this check is OK.